### PR TITLE
fix: 1.4 QA fixes after release

### PR DESCRIPTION
**Pull request by ArmanNik**
## What does this PR do?

Various fixes from QA

## Test Plan

Manual

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/console/project-[project]/functions/templates/+page.svelte [hunk 1]
```diff
@@ -82,23 +79,10 @@
...
-                    class="button is-text is-only-icon"
-                    aria-label="Clear search"
-                    style="--button-size:1.5rem;">
-                    <span class="icon-x" aria-hidden="true" />
-                </button>
-            </div>
+            <InputSearch
+                placeholder="Search templates"
+                on:clear={clearSearch}
+                on:change={applySearch} />
```
Comment (christyjacob4) [inline src/routes/console/project-[project]/functions/templates/+page.svelte:85]: why dont we need the debouncing anymore?

File: src/lib/elements/forms/inputSearch.svelte [hunk 2]
```diff
@@ -26,13 +27,19 @@
...
             const target = event.target as HTMLInputElement;
             value = target.value;
+            dispatch('change', value);
         }, debounce);
-    };
+    }
+
+    function clearSearch() {
+        value = '';
+        dispatch('clear');
```
Comment (christyjacob4) [inline src/lib/elements/forms/inputSearch.svelte:41]: do we use constants for dispatched event names in other parts of the code? If yes we should keep it consistent

**Review by christyjacob4 — APPROVED** [review]
Looks good. Please address the comments

File: src/routes/console/project-[project]/functions/templates/+page.svelte (hunk 1 above)
Comment (ArmanNik) [reply src/routes/console/project-[project]/functions/templates/+page.svelte:85]: It's already the InputSearch component

File: src/lib/elements/forms/inputSearch.svelte (hunk 2 above)
Comment (ArmanNik) [reply src/lib/elements/forms/inputSearch.svelte:41]: Usually we handle it like this :+1:

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**